src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -504,21 +507,24 @@ public void sanitizeForSaving() {
         this.courseId = Sanitizer.sanitizeTitle(courseId);
         this.creatorEmail = Sanitizer.sanitizeEmail(creatorEmail);
     }
-
+    
+    public boolean isValidJsonObject(String jsonString) {
```

Comment:
Maybe rename this to `isValidJsonString` since the input is a `String`. Also, could make this method private.